vars/getChrootName.groovy

```diff
@@ -14,7 +14,7 @@ String call(String distro) {
 
     return[ 'centos7':    'centos+epel-7-x86_64',
             'el8':        'rocky+epel-8-x86_64',
-            'leap15':     'opensuse-leap-15.3-x86_64',
+            'leap15':     'opensuse-leap-15.6-x86_64',
```

Comment:
Per our conversation, I think you are going to have to hold this back to 15.5.  See also https://github.com/rpm-software-management/mock/issues/1516.

```suggestion
            'leap15':     'opensuse-leap-15.5-x86_64',
```

Reply:
Fixed